index.html

```diff
@@ -2,232 +2,239 @@
+                  <i class="fa fa-linkedin"></i>
+                </a>
+
+                <a href="https://www.facebook.com/virtision">
+                  <i class="fa fa-facebook"></i>
+                </a>
+
+                <a href="https://www.instagram.com/virtision/">
+                  <i class="fa fa-instagram"></i>
+                </a>
+
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-5 offset-lg-1 float-right mb-3">
+          <video playsinline="" muted="" class="rounded shadow-lg video" src="images/demo.mp4" loop autoplay></video>
+        </div>
+
+      </div>
+    </div>
+  </div>
+  <div class="main">
+    <div class="container value">
+      <div class="row justify-content-center">
+        <div class="col-lg-8 center">
+          <h1 class="pop"> For any creators at any level to create and share their imagination in XR</h1>
+          <!--Toy Faces created by Amrit Pal Singh (https://amritpaldesign.com/toy-faces)-->
+        </div>
+        <div class="col-lg-12">
+          <img style="width:inherit" class="rounded shadow-lg" src="./images/artist2.png">
+        </div>
+
+      </div>
+    </div>
+    <div class="container value">
+      <div class="row pt-5">
+        <div class="col-lg-6">
+          <img class="demo" src="./images/creation.png" width="400">
+        </div>
+        <div class="col-lg-6">
+          <h1 class="pop">Simply create your XR story with just a few taps</h1>
+          <h3 class="motto">Our seamless creation experience will unblock your creativity.</h3>
+        </div>
+
+      </div>
+    </div>
+    <div class="container value">
+      <div class="row">
+        <div class="col-lg-6 order-md-2 order-sm-1">
+          <img class="demo" src="./images/sanbox.png" width="400">
+          <!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+        </div>
+        <div class="col-lg-6  order-md-1 order-sm-2">
+          <h1 class="pop">The XR sandbox for your creativity</h1>
+          <h3 class="motto">Powered by a community-built library of 3D assets, music, animations, and effects</h3>
+
+        </div>
+
+      </div>
+    </div>
+    <div class="container value">
+      <div class="row">
+        <div class="col-lg-6">
+          <img class="demo" src="./images/alldevice2.png" width="400">
+        </div>
+        <div class="col-lg-6">
+          <h1 class="pop">Share and enjoy a universe of XR creations with any device</h1>
+          <h3 class="motto">Remove barriers between your XR artworks and your audience</h3>
+
+        </div>
+      </div>
+    </div>
+    <div class="end rounded shadow-lg">
+      <div class="container value mt-5">
+        <div class="row justify-content-center">
+          <div class="col-lg-9 center-close mt-5">
+            <h1 class="pop">Bring more people into contributing to the future of XR</h1>
```

Comment:
Reads a little weird here.

Maybe:
a) "Enabling more people to contribute to the future of XR"
b) "Bringing more people to contribute to the future of XR"
c) "Helping more people with their creative journey and contributing to the future of XR"

Reply:
B is my original, the current one is from Bdang Suggestion

Reply:
i will stay with B then